CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**11.0 - 3/26/24**
+
+ - Update to 2021 data for Ethiopia 
```

Comment:
This should also include the bug-fix changes to pregnancy observation, changes to stratification, removal of the IFA scenario, and removal of background morbidity.